### DISCUSSION TITLE
[supply] Is there support for targeting Wear OS Only form factor?

### DISCUSSION THREAD

**Discussion by jikyu** [fastlane/fastlane#21323] [Q&A]
Google Play Console recently created a separate release process for Wear OS only form factors. Does `fastlane supply` support this flow?

<img width="1291" alt="Screenshot 2023-06-05 at 2 47 47 PM" src="https://github.com/fastlane/fastlane/assets/4361648/2fc80dfb-317c-4070-bea7-8c42954c2c8f">

**Comment by nrotonda** [comment]
Followed the documentation below and I'm getting the error "Track not found: wear:qa"

[https://developers.google.com/android-publisher/tracks#ff-track-name](https://developers.google.com/android-publisher/tracks#ff-track-name)

Edit: I was able to successfully upload to the Play Store using the track "wear:internal" instead of "wear:qa". I don't deploy directly to production from fastlane, but I would expect that "wear:production" would work
